### PR TITLE
Remove height minuses

### PR DESCRIPTION
**Pull request by Vidmapps**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,9 +15,9 @@
...
             </div>
             <div class="right flex-and-center">
-                <div class="right-item hover-color"><img src="img/025-mail.svg" class="-mail object-fit-contain-and-pointer"></div>
+                <div class="right-item hover-color"><img src="img/025-mail.svg" class="mail object-fit-contain-and-pointer"></div>
                 <div class="right-item hover-color"><img src="img/notif.svg" class="notif object-fit-contain-and-pointer"></div>
-                <div class="right-item hover-color"><img src="img/009-four.svg"class="-four object-fit-contain-and-pointer"></div>                                                        
+                <div class="right-item hover-color"><img src="img/009-four.svg"class="four object-fit-contain-and-pointer"></div>                                                        
                 <div class="profile flex-and-center">
                     <div class="drop-down hover-color"><i class="fas fa-chevron-down"></i></div>
                     <div class="name hover-color">Luke Borwn</div>
```
Comment (alynioke) [inline index.html]: forgot to remove minuses in .css files too

Comment (Vidmapps) [reply index.html]: Does it seem to be fine now?  I can't find anything left with minuses.

Comment (alynioke) [reply index.html]: flow how i'm checkint your PR:
i see you've changed classname in html (`clock-circular-outline`, `warning`, `move-arrows`, `screwdriver-and-wrench-crossed`). I go to CSS files to see if you've changed name of class in CSS too. Then i see - there is no such class :D So best is to remove it from HTML in that case. Beucause it doesn't make sense to refactor, but then this class doesn't work actually.

Comment (Vidmapps) [reply index.html]: true... true...  :D

Comment (Vidmapps) [reply index.html]: Should be ok now

Comment (alynioke) [reply index.html]: i still see classes refactored in HTML but nothing in css touched

Comment (Vidmapps) [reply index.html]: As per the previous talk, I should leave the class names of the img if for the future changes I will be adding some CSS? Or I should remove them and add only if CSS is required?

Comment (alynioke) [reply index.html]: it's okay to leave for future here too, if you intend to add css

**Comment by alynioke** [comment]
beware of conflicts

**Pull request merged**